benchmark/utils/general.hpp

```diff
@@ -366,8 +368,8 @@ double compute_residual_norm(const gko::LinOp *system_matrix,
 
 
 template <typename ValueType>
-double compute_max_relative_norm2(vec<ValueType> *result,
-                                  const vec<ValueType> *answer)
+ValueType compute_max_relative_norm2(vec<ValueType> *result,
```

Comment:
As usual for most of the functions/variables in this file, should these be valuetype of `remove_complex<ValueType>`?

Reply:
Yes, I did not think about that, good catch.
In that case, I also need to test if the benchmark works with complex types like `std::complex<double>`.